exercises/java-primitives/src/main/java/com/cbfacademy/StringExercises.java

```diff
@@ -3,19 +3,68 @@
 public class StringExercises {
 
     public String fromCharacters(char[] characters) {
-        // TODO - Write code that creates a String from the input array of characters
-        throw new RuntimeException("Not implemented");
+        //TODO - Write code that creates a String from the input array of characters
+
+        String sentence = new String (characters);
+        return sentence;
+        
     }
 
     public Long howMany(String text, Character character) {
         // TODO - Write code to determine how many of the input ${character} are contained in the input ${text}
-        throw new RuntimeException("Not implemented");
+
+        if (text == null || text.isEmpty() || character == null) {
+            return 0L;
+        }
+    
+        long count = 0L;
+        for (int i = 0; i < text.length(); i++) {
+            if (text.charAt(i) == character) {
+                count++;
+            }
+        }
+    
+        return count;
     }
+        
+       
+    
 
     public Boolean isPalindrome(String word) {
-        // TODO - Write code to determine whether the input ${word} is a palindrome
-        throw new RuntimeException("Not implemented");
-    }
+        /* TODO - Write code to determine whether the input ${word} is a palindrome */
+
+        // Initializing an empty string to store the reverse
+        // of the original string
+        String reverseString = "";
+
+        // Initializing a new boolean variable for the
+        // answer
+        boolean answer = false;
+
+        for (int i = word.length() - 1; i >= 0; i--) {
+            reverseString = reverseString + word.charAt(i);
```

Comment:
You could use an addition assignment operator here:

`reverseString = reverseString + word.charAt(i)` is the same as `reverseString += word.charAt(i)`;